autoload/vista/popup.vim

```diff
@@ -9,6 +9,9 @@ let s:popup_delay = get(g:, 'vista_floating_delay', 100)
 function! s:ClosePopup() abort
   if exists('s:popup_winid')
     call popup_hide(s:popup_winid)
```

Comment:
The reason for using `popup_hide` is to reuse the popup. Use `popup_close()` if you want to really close the popup. But I'm actually unwilling to do this as it will explode the buffer number.

Reply:
OK, I'll play with it for a bit and see if I can do it another way.